config/providers/liquid.rb

```diff
@@ -4,7 +4,11 @@
   prepare { require "liquid" }
 
   start do
-    default = Liquid::Environment.build { it.error_mode = :strict }
+    # Register custom filters
+    default = Liquid::Environment.build do |env|
```

Comment:
:art: Can you use `environment` instead of `env` since `environment` is consistent with the `renderer` lambda below.